packages/flame/test/game/component_rendering_test.dart

```diff
@@ -79,9 +79,9 @@ void main() {
         await game.ensureAddAll([
           _MyComponent(4),
           _MyComponent(1),
-          _MyComponent(2, respectCamera: false),
-          _MyComponent(5, respectCamera: false),
-          _MyComponent(3, respectCamera: false),
+          _MyComponent(2, positionType: PositionType.viewport),
+          _MyComponent(5, positionType: PositionType.viewport),
+          _MyComponent(3, positionType: PositionType.viewport),
           _MyComponent(0),
         ]);
 
```

Comment:
can be a follouwp, but would be good to have a test for the widget case. maybe even an example so we can make sure it works as we are envisioning

Reply:
Added tests, but no example yet!

Reply:
Tried it though, and it works like it should. :)